nima/webserver/webserver/src/test/java/io/helidon/nima/webserver/builder/ContentEncodingContextTest.java

```diff
@@ -0,0 +1,95 @@
+package io.helidon.nima.webserver.builder;
+
+import io.helidon.common.http.Headers;
+import io.helidon.common.http.Http;
+import io.helidon.nima.http.encoding.ContentDecoder;
+import io.helidon.nima.http.encoding.ContentEncoder;
+import io.helidon.nima.http.encoding.ContentEncodingContext;
+import io.helidon.nima.webclient.http1.Http1Client;
+import io.helidon.nima.webclient.http1.Http1ClientResponse;
+import io.helidon.nima.webserver.WebServer;
+import io.helidon.nima.webserver.http.Handler;
+import io.helidon.nima.webserver.http.HttpRouting;
+import io.helidon.nima.webserver.http.ServerRequest;
+import io.helidon.nima.webserver.http.ServerResponse;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.NoSuchElementException;
+import java.util.function.Consumer;
+
+public class ContentEncodingContextTest {
```

Comment:
The class can be package local

Reply:
This test should probably be in `nima/tests/integration/webserver` and use `@ServerTest`. There is an annotation `@SetUpServer` that allows you to customize the server builder